src/classes/BDI_DataImport_TEST2.cls

```diff
@@ -1384,21 +1384,27 @@ private with sharing class BDI_DataImport_TEST2 {
     *    nothing imported
     *    records marked with fls error
     **********************************************************************************************************/            
-    static testMethod void testFieldLevelSecurity() {
+    static testMethod void givenUserHasOnlyReadAccessWhenUsingHelpTextMapThenImportsShouldFail() {
```

Comment:
Since `testMethod` is deprecated in favor of `@IsTest`, if there aren't too many occurrences in this class it's best to update the various test methods since the class is being modified.